packages/design-system/src/interfaces/forms.d.ts

```diff
@@ -183,6 +183,14 @@ export interface ISearchBarProps extends ISearchBarHtmlProps {
   onExpand?(): void;
 }
 
+export interface IIconProps extends React.HTMLAttributes<HTMLSpanElement> {
+  source: string;
+  size?: string;
+  iconType?: string;
```

Comment:
use correct type here:
`export type IconType = 'primary' | 'inverted' | 'link' | 'success' | 'warning' | 'error'`
The same goes for size, let's declare some type instead of using just string - it doesn't tell anything to developer